### DISCUSSION TITLE
keyball61 トラックボール基盤のジャンパのはんだ付け失敗

### DISCUSSION THREAD

**Discussion by tanakabeyblade** [Yowkees/keyball#243] [Q&A]
当方電子工作は初心者です。
一通り組みおわったもののトラックボールが反応せず、ジャンパ部分を温め直していたところ、パッドが剥離したのかはんだが乗らなくなってしまいました。この際の対処方法をご教示いただけますか？よろしくお願いいたします。
![PXL_20230529_155039664](https://github.com/Yowkees/keyball/assets/134957725/92255866-cd09-4ea0-a194-52de4496c693)

**Comment by tanakabeyblade** [comment]
上記の件には無関係かもしれませんが、USB接続をすると両方のpro microが赤く光ります。ファームウェアはガイドにあったものを使っていますが、これは正常な動作ですか？

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
ProMicroが赤く光るのは正常です。
組み立て中のKeyballの種類は何でしょうか？（39 or 44 or 61 ?)

**Reply by tanakabeyblade** [reply]
説明が不明瞭ですみませんでした、こちらkeyball 61です。トラックボールは右手で組み立てています。

**Reply by tanakabeyblade** [reply]
![PXL_20230607_003533457](https://github.com/Yowkees/keyball/assets/134957725/d46c68a5-c957-4fad-87fa-35cfd6ec79e3)

過去の投稿を参考にしてみましたが、こちらのBall PCBとProMicroのMISO/MOSIを被覆銅線で繋げればいいのでしょうか？
よろしくお願いいたします

**Comment by Yowkees** [comment]
![Keyball61_ballfix](https://github.com/Yowkees/keyball/assets/84209772/d0961805-0354-48ad-aaec-3a337ac4f80f)
お写真ありがとうございます。
少々距離が遠いですが、添付画像の同色どうしをエナメル線やゴム被覆導線で接続してください。
これでジャンパしたのと同じ回路となり、トラックボールを復活できます。
お手数ですがご対応お願いいたします。
